std_msgs/Header header

float64 u_ref
float64 v_ref
float64 yaw_ref
float64 K1
float64 K2
float64 Ka
float64 sigma
uint8 state
float64 heading_ref

float64 cross_track
float64 along_track
float64 sigma_dot
float64 yaw_correction
float64 yaw_d
float64 u
float64 U